tests/tally/generator.rs

```diff
@@ -47,7 +49,7 @@ fn account_from_slice<P>(
     }
 }
 
-impl VoteRoundGenerator {
+impl<'a> VoteRoundGenerator {
```

Comment:
I'm not sure why this annotation is needed, I think the elision rules are enough here.